actions/pacman_action.go

```diff
@@ -0,0 +1,40 @@
+/*
+Pacman Action
+
+Install packages and their dependencies to the target rootfs with 'pacman'.
+
+Yaml syntax:
+ - action: pacman
+   packages:
+     - package1
+     - package2
+
+Mandatory properties:
+
+- packages -- list of packages to install
+*/
+package actions
```

Comment:
wondering if we should have an implementation for `Verify` which checks if pacman (and other used) commands are present on the system; if not we should bail out early?

Reply:
I think (from the pacstrap PR) we wanted to make that a separate follow-up. I have the code, but perhaps doesn't belong in this PR?